src/ContainerFactory.ts

```diff
@@ -73,6 +75,8 @@ export function createContainer(): Container {
 
     container.bind(CompaniesHouseSDK).toConstantValue(CompaniesHouseSDK(getEnvOrThrow('API_URL')));
 
+    container.bind(PenaltyIdentifierSchemaFactory).toConstantValue(new PenaltyIdentifierSchemaFactory(getEnvOrDefault('ALLOWED_COMPANY_PREFIXES', '*')));
```

Comment:
can you split this over two lines for readability? it's quite a bit past the right hand margin in the IDE ;)

Reply:
Also what does the default `*` give us? Could just throw an exception